src/components/calculator.js

```diff
@@ -1,31 +1,53 @@
+          { operation}
+          { next }
+        </div>
         <div className="cal-buttons">
-          <button type="button" className="btn">AC</button>
-          <button type="button" className="btn">+/-</button>
-          <button type="button" className="btn">%</button>
-          <button type="button" className="btn primary">/</button>
-          <button type="button" className="btn">7</button>
-          <button type="button" className="btn">8</button>
-          <button type="button" className="btn">9</button>
-          <button type="button" className="btn primary">*</button>
-          <button type="button" className="btn">4</button>
-          <button type="button" className="btn">5</button>
-          <button type="button" className="btn">6</button>
-          <button type="button" className="btn primary">-</button>
-          <button type="button" className="btn">1</button>
-          <button type="button" className="btn">2</button>
-          <button type="button" className="btn">3</button>
-          <button type="button" className="btn primary">+</button>
-          <button type="button" className="btn zero">0</button>
-          <button type="button" className="btn">.</button>
-          <button type="button" className="btn primar">=</button>
+          <button type="button" className="btn" onClick={this.handleClick}>AC</button>
+          <button type="button" className="btn" onClick={this.handleClick}>+/-</button>
+          <button type="button" className="btn" onClick={this.handleClick}>%</button>
+          <button type="button" className="btn primary" onClick={this.handleClick}>÷</button>
+          <button type="button" className="btn" onClick={this.handleClick}>7</button>
+          <button type="button" className="btn" onClick={this.handleClick}>8</button>
+          <button type="button" className="btn" onClick={this.handleClick}>9</button>
+          <button type="button" className="btn primary" onClick={this.handleClick}>x</button>
+          <button type="button" className="btn" onClick={this.handleClick}>4</button>
+          <button type="button" className="btn" onClick={this.handleClick}>5</button>
+          <button type="button" className="btn" onClick={this.handleClick}>6</button>
+          <button type="button" className="btn primary" onClick={this.handleClick}>-</button>
+          <button type="button" className="btn" onClick={this.handleClick}>1</button>
+          <button type="button" className="btn" onClick={this.handleClick}>2</button>
+          <button type="button" className="btn" onClick={this.handleClick}>3</button>
+          <button type="button" className="btn primary" onClick={this.handleClick}>+</button>
+          <button type="button" className="btn zero" onClick={this.handleClick}>0</button>
+          <button type="button" className="btn" onClick={this.handleClick}>.</button>
+          <button type="button" className="btn primary" onClick={this.handleClick}>=</button>
```

Comment:
- [ ] Kindly create another component as a child of the `Calculator` component and pass the component data as props to the child component. You could put all the buttons in an array, map through the array and pass each button as a prop to the component. You can read this article to have a better understanding https://www.geeksforgeeks.org/how-to-map-data-into-components-using-reactjs/